.travis.yml

```diff
@@ -1,16 +1,54 @@
 language: python
+
 python:
   - 3.6
 
+os:
+  - linux
+  - linux-ppc64le
+
+env:
+  - DOCKERFILE=Dockerfile IMAGE=max-base
+  - DOCKERFILE=Dockerfile.pai IMAGE=max-base-powerai
+
+matrix:
+  exclude:
+  - env: DOCKERFILE=Dockerfile IMAGE=max-base
+    os: linux-ppc64le
+
+dist: xenial
+
 services:
   - docker
 
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux-ppc64le" ]]; then
+       ARCH=ppc64le;
+    fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+       ARCH=x86_64;
+    fi
+  - mkdir -p ~/.docker
+  - echo '{"experimental":"enabled"}' | tee ~/.docker/config.json 
+  - sudo service docker restart
+
 install:
-  - docker build -t max-base .
+  - docker build -f "$DOCKERFILE" -t "$REPO"/"$IMAGE":"$ARCH"-latest .
```

Comment:
That is fine with me. I wrote it this way to keep the powerai-based image entirely separate from the vanilla one, but we can definitely put them in the same repo. Would you prefer something like max-base:powerai-<arch>-latest ?

The only real requirement is to make sure people know what they're getting. There won't be a non-powerai based image for ppc64le (i.e. pulling max-base:latest won't get you a ppc64le image, even on a ppc64le machine), so that needs to be clear in the documentation at the very least.

Reply:
Based on our slack convo, this only needs minor changes. Give the max-base portion of this file is solely for testing, it's name in travis doesn't matter, for the power side of things this should probably be changed from `"$REPO"/"$IMAGE":"$ARCH"-latest` to `codait/max-base:"powerai-$ARCH"-latest` see my example code block in my PR comment below for my example way of replacing this